docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #127.    def DAMP(T, m, split_index):
</pre><p>Later, we may add the param <code>T_subseq_isconstant</code> to this function </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
It is important to be aware of (close to) constant subsequences and handle them before computing the discord. 

Also, see the note in the MATLAB code shareed by the authors of DAMP: https://drive.google.com/file/d/1EPDhFXQ2goTJ5m_x1S84-KNpQFsRNVmf/view